### DISCUSSION TITLE
Use Plugin in ionic 6

### DISCUSSION THREAD

**Discussion by jhanzaibkhadim199** [EMI-INDO/emi-indo-cordova-plugin-admob#18] [Q&A]
How to use this plugin with ionic 6 ? Also can anybody confirms that this plugin support to use google ads anywhere in my view in mobile app e.g. ads can be placed between a list of elements.

**Comment by EMI-INDO** [comment]
@jhanzaibkhadim199 

[Documentation Capacitor example: ](https://github.com/EMI-INDO/emi-indo-cordova-plugin-admob/discussions/29)
